### PR TITLE
Add content and basic style to the page

### LINKED ISSUE #1: Initialize Application
**As a user I want to see the home page when I load the site**

> Assigned to Miroslav

### REPO

- This user story is developed on branch `page-structure`.
- This branch is merged to `master` branch after completion.

### Changes in HTML

- Write the initial content for the page
- Write all the buttons needed for the page

### Changes in CSS

- Make starter basic style to the content the page

### Changes in JavaScript

- Write all questions, possible answers, correct answers and links in the "questions" array in the data.js file.

### PR DESCRIPTION
**Pull request by miroslavveljanoski**
Hi guys, I made the first part of our page.

Please review my code and let me know if I need to change something

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -19,50 +19,73 @@ const state = {
...
-          href: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof'
+          text: 'javascript.info',
+          href: 'https://javascript.info/object',
         },
-      ]
+      ],
+    },
+    {
+      text: 'Who is the best teacher for web development?',
+      answers: ['Evan Cole', 'Mosh Hamedani', 'Wes Bos', 'Brad - Traversy Media'],
```
Comment (aleks2407) [inline src/data.js]: I think, we should keep 3 options of answers for all questions. We have only 3 buttons in index.html

Comment (miroslavveljanoski) [reply src/data.js]: I fix this problem @aleks2407

Comment (aleks2407) [reply src/data.js]: thank you!

**Review by oguzkarademir — APPROVED** [review]

**Review by Feruzteame — APPROVED** [review]

**Comment by Feruzteame** [comment]
great! thank you @miroslavveljanoski

**Review by aleks2407 — APPROVED** [review]

**Pull request merged**